__tests__/Itinerary.test.js

```diff
@@ -0,0 +1,20 @@
+const Port = require('../src/Port.js');
+const Ship = require('../src/Ship.js');
+const Itinerary = require('../src/Itinerary.js');
+
+describe('Itinerary', () => {
+    it('can be instansiated', () => {
+        const liverpool = new Port('Liverpool')
+        const dublin = new Port('Dublin');
+        const itinerary = new Itinerary([liverpool, dublin]);
+        expect(itinerary).toBeInstanceOf(Object) 
```

Comment:
You can be a bit more specific and check if itinerary is an instance of Itinerary.
This is correct anyway, but it's a minor improvement.